archbuild/local_build/build.sh

```
@@ -37,7 +37,7 @@ readonly ARCH_BUNDLE_NAME
 # ask for deb build
 read -p "rebuild deb package? (y/n): " rebuild
 if [[ "$rebuild" =~ ^[Yy]$ || -z $rebuild ]]; then
-  pnpm build -c $PROJECT_ROOT_DIR/src-tauri/tauri.linux.conf-local.json -b deb
+  pnpm build -c $PROJECT_ROOT_DIR/src-tauri/tauri.conf-local.json -b deb
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)